unity-renderer/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs

```diff
@@ -254,7 +254,7 @@ public void Dispose()
         /// <param name="onLoadComplete">Callback function for when load is completed</param>
         /// <param name="token">Cancellation token</param>
         /// <returns></returns>
-        public async UniTask LoadScene(CancellationToken token, int sceneIndex = -1, bool showSceneObj = true)
+        public async Task LoadScene(CancellationToken token, int sceneIndex = -1, bool showSceneObj = true)
```

Comment:
Couldn't find a way to synchronously wait for a result for `UniTask` so I decided to use a `Task` instead since I can do `Task.Wait(timeout)` wich is needed for this kind of editor scirpts